.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
+
+<!-- Please describe clearly and concisely what the bug is. -->
+
+## Steps to reproduce
+
+<!-- Please provide detailed steps on how to reproduce the bug. -->
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## Screenshots
```

Comment:
Having a URL where the issue can be seen on the frontend is ideal, whenever possible. So there be some prompt for the user to provide a URL.

Otherwise, if they can't provide the URL, they should go to “View source” in the browser and copy all to paste in a Gist, and then share that.